samples/dicom-demo-env/environments/cloud/deploy-cloud-environment.bicep

```diff
@@ -0,0 +1,124 @@
+@description('Specifies the location for resources.')
+param location string = resourceGroup().location
+
+@description('Name of the AHDS workspace. This will appear in the API URL.')
+param workspace_name string = 'workspace${uniqueString(resourceGroup().id)}'
+
+@description('Name of the AHDS DICOM service. This will appear in the API URL.')
+param dicom_service_name string = 'dicom${uniqueString(resourceGroup().id)}'
+
+@description('Name of the AHDS FHIR service. This will appear in the API URL.')
+param fhir_service_name string = 'fhir${uniqueString(resourceGroup().id)}'
+
+@description('Should deploy FHIR')
+param should_deploy_fhir bool = false
+
+@description('Should deploy DICOM')
+param should_deploy_dicom bool = true
+
+// @description('Your existing DICOM service URL (format: https://yourworkspacename-yourdicomservicename.dicom.azurehealthcareapis.com)')
+// param dicomServiceUrl string = 'https://sjbrsnadicomws-rsnadicom.dicom.azurehealthcareapis.com'
+
+@description('Your existing Azure AD tenant ID (format: 72xxxxxf-xxxx-xxxx-xxxx-xxxxxxxxxxx)')
+@secure()
+param aadTenantId string 
+
+@description('Your existing Application (client) ID (format: 1f8xxxxx-dxxx-xxxx-xxxx-9exxxxxxxxxx)')
+@secure()
+param applicationClientId string 
+
+@description('Your existing Application (client) secret')
+@secure()
+param applicationClientSecret string 
+
+// This is VERY confusing! And you can't get the Principal Object ID from the Portal UI. 
```

Comment:
NIT: This is wrong, you can get the principal from the Portal.

Reply:
Yeah, a lot of those comments are outdated.  The App registration steps explain how to get the ID from the portal 😄